cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/ShellCommand.java

```diff
@@ -47,6 +48,9 @@
     @CommandLine.Option(names = {"-t", "--time-out"}, description = "Timeout to use for tree parsing.")
     private long timeOut = 100;
 
+    @CommandLine.Option(names = {"-o", "--open-file"}, description = "Open file and load initial declarations.")
```

Comment:
Shouldn't  the option simply be something like `--file`? IMO, --open-file kinda implies a boolean option.

Reply:
Sure, but the command argument will have to be `-o` still because `-f` was taken for `--force-dumb`
So maybe this should be named to a different name?

Reply:
IMO we don't need a short option for every single option. Can we use `-f` for --file and remove it from --force-dumb? If not maybe we can remove the short option here? WDYT?

Reply:
I think removing the short option from `--force-dumb` might be better, there not much use case for that option

Reply:
+1, let's also mention this in the release note.